### PR TITLE
New Ambient capability flag(since Linux 4.3) "CapAmb:	000000000000000…

### PR DESCRIPTION
**Pull request by sudeeshjohn**
New Ambient capability flag(since Linux 4.3) "CapAmb:	000000000000000… Changes made to pick ppc64le fdinfo flags. Also the failure reporting method.

Signed-off-by: Sudeesh John <sudeesh@linux.vnet.ibm.com>

### CODE REVIEW

File: libvirt/tests/src/conf_file/qemu_conf/clear_emulator_capabilities.py [hunk 1]
```diff
@@ -28,6 +26,7 @@ def get_qemu_process_caps(vm):
         CapPrm: 0000001fffffffff
         CapEff: 0000001fffffffff
         CapBnd: 0000001fffffffff
+        CapAmb:	0000000000000000
```
Comment (balamuruhans) [inline libvirt/tests/src/conf_file/qemu_conf/clear_emulator_capabilities.py]: there is a tab after colon.

**Comment by balamuruhans** [comment]
As we have replaced `error.TestNAError` to `test.fail()`, let us use the same in doc string in,
`L 34: :raise TestNAError: If can not get capabilities from proc.`

File: libvirt/tests/src/conf_file/qemu_conf/clear_emulator_capabilities.py [hunk 2]
```diff
@@ -84,14 +83,14 @@ def get_qemu_process_caps(vm):
         # Restart libvirtd to make change valid.
         if not libvirtd.restart():
             if expected_result != 'unbootable':
-                raise error.TestFail('Libvirtd is expected to be started '
-                                     'with clear_emulator_capabilities = '
-                                     '%s' % clear_emulator_capabilities)
+                raise test.fail('Libvirtd is expected to be started '
```
Comment (balamuruhans) [inline libvirt/tests/src/conf_file/qemu_conf/clear_emulator_capabilities.py]:
```
`raise` is not required here and else where,
test.fail('Libvirtd is expected to be started '
          with clear_emulator_capabilities = '
          '%s' % clear_emulator_capabilities)
```

File: libvirt/tests/src/conf_file/qemu_conf/clear_emulator_capabilities.py [hunk 3]
```diff
@@ -48,11 +47,11 @@ def get_qemu_process_caps(vm):
         finally:
             proc_stat_fp.close()
 
-        if len(caps) == 4:
+        if len(caps) == 5:
             return caps
         else:
-            raise error.TestNAError('Host do not support capabilities or '
-                                    'an error has occured.')
+            raise test.cancel('Host do not support capabilities or '
```
Comment (balamuruhans) [inline libvirt/tests/src/conf_file/qemu_conf/clear_emulator_capabilities.py]: @sudeeshjohn, got missed here.

Comment (sudeeshjohn) [inline libvirt/tests/src/conf_file/qemu_conf/clear_emulator_capabilities.py]: hmm..yeah..sorry for being so careless :(

**Comment by balamuruhans** [comment]
@sudeeshjohn, Thank you

**Pull request merged**